tools/challenge-helper-scripts/create-project.ts

```diff
@@ -375,11 +375,11 @@ void getAllBlocks()
       {
         name: 'position',
         message: 'At which position does this appear in the module?',
-        default: 1,
+        default: 0,
```

Comment:
Instead of allowing 0, I think we should make the script to accept 1-based values but evaluate that as 0-based. This will maintain the consistency, as the `order` prompt right after this one expects a 1-based value.

Here is an example:

https://github.com/freeCodeCamp/freeCodeCamp/blob/745d4632530636531d8aa9793cfc55f44a391d14/tools/challenge-helper-scripts/create-language-block.ts#L121-L126